Kickstarter-iOS/Views/Controllers/ManagePledgePaymentMethodView.swift

```diff
@@ -133,9 +133,6 @@ private let paymentMethodAdaptableStackViewStyle: StackViewStyle = { stackView i
 
 private let titleLabelStyle: LabelStyle = { label in
   label
-    |> \.textColor .~ UIColor.black
-    |> \.font .~ UIFont.ksr_subhead().bolded
-    |> \.adjustsFontForContentSizeCategory .~ true
+    |> checkoutTitleLabelStyle
```

Comment:
On the other hand `titleLabelStyle` is identical to the one we've been using in other places.